cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/generator/InputInterpreterImpl.java

```diff
@@ -104,8 +104,9 @@ private Object readInput(Path path, Charset inputCharset, Map<TriggerInterpreter
           "Was not able to read input {} with inputreader '{}' although it was reported to be most likely readable. Trying next input reader...",
           path, triggerInterpreter, e);
     } catch (Throwable e) {
-      LOG.debug("While reading the input {} with the inputreader {}, an exception occured. Trying next input reader...",
-          path, triggerInterpreter, e);
+      LOG.warn("While reading the input {} with the inputreader {}, an exception occured: {} Trying next input reader...",
+              path, triggerInterpreter, e.getMessage());
+      if (LOG.isDebugEnabled()) e.printStackTrace();
```

Comment:
Don't print stacktraces on your own, pass the e on case of debug as last log parameter